Gemfile

```diff
@@ -46,10 +46,7 @@ end
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> anywhere in the code.
   gem 'web-console', '>= 3.3.0'
-  gem 'listen', '~> 3.0.5'
```

Comment:
I actually think I'm going to offer this as a stand-alone pull request, if you guys want to go along with it. It makes life so much easier!